frontend/src/pages/RadarPage.js

```diff
@@ -57,6 +57,17 @@ function RadarPage() {
   const { getTechRadarData, getCsvData } = useData();
   const getTechnologyStatus = useTechnologyStatus();
 
+  const [selectedDirectorate, setSelectedDirectorate] =
+    useState('Digital Services');
+
+  const directorateOptions = ['Digital Services', 'Data Science', 'DGO'];
+
+  const directorateColourMap = {
+    'Digital Services': '#1f77b4', // Blue
+    'Data Science': '#ff7f0e', // Orange
+    DGO: '#2ca02c', // Green
```

Comment:
Think we can put them as Data Growth and Operations (DGO) and also abbr. Digital Services (DS) and Data Science Campus (DSC)